config.py

```diff
@@ -737,7 +737,7 @@ def create_transaction(db):
     WEBHOOK_HOSTNAME_BLACKLIST = []
 
     # The root URL for documentation.
-    DOCUMENTATION_ROOT = "https://access.redhat.com/documentation/en-us/red_hat_quay/3.4/"
+    DOCUMENTATION_ROOT = "https://access.redhat.com/documentation/en-us/red_hat_quay/3/"
```

Comment:
Should this be 3.5?

Reply:
`/3/`always points to the latest release so I've been moving links there so they don't need to be changed on every y-stream.